README.md

```diff
@@ -1,2 +1,68 @@
+
+**Event Trigger**:
+
+- `pull_request_target`
+- `issue-comment`
+
+**Action inputs**:
+
+| Name                                   | Description                                                                | Default                                                                                                                                                    |
+|----------------------------------------|----------------------------------------------------------------------------|------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `tos-location`                         | Terms Of Service location (URL or relative file)                           |                                                                                                                                                            |
+| `github-token`                         | Github token                                                               |                                                                                                                                                            |
+| `signature-branch`                     | *Optional*: Git branch where the signature file will be stored             | `main`                                                                                                                                                     |
+| `signature-location`                   | *Optional*: Signature file location                                        | `.github/gradle-enterprise-tos.json`                                                                                                                       |
+| `pr-comment-tos-approval-missing`      | *Optional*: PR comment added when Terms of Service are not approved        | `Please accept [Gradle Enterprise Terms Of Service](<tos-location>) to get your PR Build Scan published by commenting this PR with the following message:` |
+| `pr-comment-tos-approval-request`      | *Optional*: PR comment to approve the Terms of Service                     | `I have read Gradle Enterprise Terms Of Service and I hereby accept the Terms`                                                                             |
+| `pr-comment-tos-approval-confirmation` | *Optional*: PR comment added when Terms of Service are approved            | `All Contributors have accepted Gradle Enterprise Terms Of Service.`                                                                                       |
+| `white-list`                           | *Optional*: CSV List of users not required to approve the Terms of Service |                                                                                                                                                            |
+
+**Usage**:
+
+```yaml
+name: Gradle - Terms of Service approval verification
+
+on:
+  issue_comment:
```

Comment:
Can you add a comment that this is the event fired for PR comments? I was initially confused.

Reply:
Sure